JSDemos/Demos/Charts/CustomAnnotations/description.md

```diff
@@ -1 +1,17 @@
-To customize an annotation, declare the SVG markup in the annotation [template](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#template). Inside the template, you can access the annotation being customized. 
\ No newline at end of file
+Annotations are designed to show additional information about visualized data. You can display images, text blocks, and custom content in annotations. To create custom annotations, specify the following Chart properties.
+
+## annotations[]
+
+The [annotations](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/) array specifies a collection of annotations. This demo uses a data source array to create this collection. Each object has an [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) to make the component display annotations relative to specific arguments and a [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object that stores data from a data source.
```

Comment:
> to make the component display annotations relative to specific arguments

This phrase sounds very vague.

Also, this sentence is quite long and difficult to read and understand per se.

Did I get this right? Does it make sense?

```suggestion
The [annotations](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/) array specifies a collection of annotations. This demo uses a data source array to create this collection. Each object has the [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) property and a [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object. The [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) property positions an annotation at a specific argument value, and the [data](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#data) object stores data from a data source.
```

Reply:
I would not say "argument value" since in chart terminology arguments and values are different things.
If I say simply
> The [argument](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#argument) property positions an annotation at a specific argument, and the data...
Is it okay?

Reply:
Yeah, this will do, I guess.